### PR TITLE
D8CORE-1873: Wrapping bug for the multi-row

### PR DESCRIPTION
**Pull request by jenbreese**
# READY FOR REVIEW

# Summary
- This is to fix wrapping elements

# Needed By (Date)
- Tuesday

# Urgency
- Critcal

# Steps to Test

1. Pull this branch. Make sure you have the updates to have multi-row.
2. Generate some content. (lando drush spstc)
3. Verify that the media elements at 4 across are staying 4 across.

# Affected Projects or Products
- Needs the multi per row.

# Associated Issues and/or People
-  D8CORE-1873

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by imonroe** [comment]
So, would this count as a hotfix, since we've already done the "release" for this repo?  If so, we should tag this PR as "patch", merge this PR into `master`, and let the machinery put it out as a new patch release.  It'll get merged back into `8.x-1.x` automatically.

**Comment by sherakama** [comment]
> So, would this count as a hotfix, since we've already done the "release" for this repo? If so, we should tag this PR as "patch", merge this PR into `master`, and let the machinery put it out as a new patch release. It'll get merged back into `8.x-1.x` automatically.

Sounds like a plan to me.

File: modules/stanford_profile_drush/stanford_profile_drush.info.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: "Stanford Profile Drush"
 description: A collection of Drush commands since commands in the profile are not discoverable.
-version: 8.x-1.7
+version: 8.x-1.8-dev
```
Comment (pookmish) [inline modules/stanford_profile_drush/stanford_profile_drush.info.yml:3]:
```suggestion
version: 8.x-1.8
```

File: modules/stanford_profile_styles/stanford_profile_styles.info.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: "Stanford Profile Styles"
 description: A module for theming
-version: 8.x-1.7
+version: 8.x-1.8-dev
```
Comment (pookmish) [inline modules/stanford_profile_styles/stanford_profile_styles.info.yml:3]:
```suggestion
version: 8.x-1.8
```

File: stanford_profile.info.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 name: Stanford Profile
 description: Jumpstart Website Profile
-version: 8.x-1.7
+version: 8.x-1.8-dev
```
Comment (pookmish) [inline stanford_profile.info.yml:3]:
```suggestion
version: 8.x-1.8
```

**Comment by pookmish** [comment]
the changes do fix the issue.

**Pull request merged**